CHANGELOG

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Dec  5 15:38:36 UTC 2016 - joseivanlopez@gmail.com
+
+- fix deletion of augeas elements
```

Comment:
and please be more verbose in changes. Maybe something like `fix regression when passing nil to AugeasTree#delete`